.gitmodules

```diff
@@ -2,3 +2,6 @@
 [submodule ".github/actions/always-upload-cache"]
 	path = .github/actions/always-upload-cache
 	url = https://github.com/pat-s/always-upload-cache.git
+[submodule "themes/docsy"]
+	path = themes/docsy
+	url = https://github.com/google/docsy.git
```

Comment:
It is required to add this submodule?

Reply:
![Screenshot 2024-02-26 205345](https://github.com/apache/dubbo-website/assets/74211043/c95fff36-cd61-4d37-a788-33b649268668)

when i remove this error shows up